ui/common/src/commonMain/kotlin/social/androiddev/common/composables/Spacers.kt

```diff
@@ -0,0 +1,64 @@
+/*
+ * This file is part of MastodonX.
+ *
+ * MastodonX is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+ *
+ * MastodonX is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with MastodonX. If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package social.androiddev.common.composables
+
+import androidx.compose.foundation.layout.ColumnScope
+import androidx.compose.foundation.layout.RowScope
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.size
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun RowScope.WeightSpacer(modifier: Modifier = Modifier) {
+    Spacer(modifier = modifier.weight(1f))
+}
+
+@Composable
+fun ColumnScope.WeightSpacer(modifier: Modifier = Modifier) {
+    Spacer(modifier = modifier.weight(1f))
+}
+
+@Composable
+fun XSmallSpacer(modifier: Modifier = Modifier) {
```

Comment:
I personally don't see anything wrong with using hardcoded DP values in compose for spacing instead of wrapping them in another composable, as long as the values conform to the 4 dp material grid sizing.

Reply:
Keep in mind that this is a cross-platform project :) 
Hard-coded dimensions are not an option, even for a simple Android app, if you want it to be compatible with all kinds of devices.